File: API/Entities/StoryComment.cs
using System;
using System.Collections.Generic;
using System.Collections.ObjectModel;

namespace API.Entities
{
    public class StoryComment
    {
        public int Id { get; set; }
        public int StoryId { get; set; }
        public Story Story { get; set; }
        public int? ChapterId { get; set; }
        public int UserPostId { get; set; }
        public AppUser UserPost { get; set; }
        public int? ParentId { get; set; }
        public DateTime Created { get; set; } = DateTime.UtcNow;
        public string Content { get; set; }
        public virtual ICollection<LikeComment> Liked { get; set; }

        public StoryComment()
        {
            Liked = new Collection<LikeComment>();
        }

    }
}

File: client/src/app/_models/storycomment.ts

export interface StoryComment{
    id:number;
    storyId:number;
    parentId?:number;
    chapterId?:number;
    created:Date;
    content:string;
    userName:string;
    knownAs:string;
    point:number;
    title:string;
    image:string;
    liked:string[];
    /**
     *
     */
    // constructor() {
    //     this.id=0;
    //     this.storyId=0;
    //     this.content="";
    //     this.knownAs="";
    //     this.image="";
    // }
}